updatecli/policies/open-release-pr/updatecli.d/open-release-pr.yaml

```diff
@@ -0,0 +1,164 @@
+
+  updateCargoTomlVersion:
+    name: update Cargo.toml with new version
+    scmid: "policy-repo"
+    disablesourceinput: true
+    dependson:
+      - condition#checkCargoTomlExists
+    kind: shell
+    spec:
+      shell: /usr/bin/bash
+      command: |
+        export new_version='{{ source "newestGitHubDraftReleaseVersion" }}'
+        awk -v new_version="$new_version" '
+          BEGIN { in_package = 0 }
+          /^\[package\]/ { in_package = 1 }
+          in_package && /^version = / { sub(/version = ".*"/, "version = \"" new_version "\"") }
+          /^\[/ && !/^\[package\]/ { in_package = 0 }
+          { print }
+        ' Cargo.toml > Cargo.toml.tmp && mv Cargo.toml.tmp Cargo.toml
+      changedif:
+        kind: file/checksum
+        spec:
+          files:
+            - Cargo.toml
+
+  updateCargoLockVersion:
+    name: update Cargo.lock with new version
+    scmid: "policy-repo"
+    disablesourceinput: true
+    dependson:
+      - condition#checkCargoLockExists
+      - target#updateCargoTomlVersion
+    kind: shell
+    spec:
+      shell: /usr/bin/bash
+      command: |
+        export package_name='{{ source "cargoPackageName" }}'
+        export new_version='{{ source "newestGitHubDraftReleaseVersion" }}'
+        # note: if testing locally, copy the awk command without trailing spaces
+        awk -v new_version="$new_version" -v package_name="$package_name" '
+          BEGIN { package_found = 0 }
+          /^\[\[package\]\]/ { package_found = 0 }
+          $0 ~ "^name = \"" package_name "\"$" { package_found = 1 }
+          package_found && /^\s*version = / { sub(/version = ".*"/, "version = \"" new_version "\"") }
+          { print }
+        ' Cargo.lock > Cargo.lock.tmp && mv Cargo.lock.tmp Cargo.lock
+      changedif:
+        kind: file/checksum
+        spec:
+          files:
+            - Cargo.lock
+
+actions:
+  openUpdatePR:
+    title: 'build: Prepare for release {{ source "newestGitHubDraftReleaseVersion" }}'
+    kind: "github/pullrequest"
+    scmid: "policy-repo"
+    spec:
+      automerge: {{ .pr.automerge }}
+      mergemethod: squash
+      description: |
+        Automatic bump of versions to prepare for a release with version `{{ source "newestGitHubDraftReleaseVersion" }}`.
+        Merging this PR will trigger a release of this policy.
+
+        Changes since last released tag: (if no tag, since HEAD^)
+        https://github.com/{{ .github.owner }}/{{ .github.repo }}/compare/{{ source "newestReleaseTagVersion" | default "HEAD^" }}...updatecli_main_release_pr
+
+        > [!NOTE]
+        > This PR has been created by automation.
+      draft: {{ .pr.draft }}
+#     {{ if .pr.labels }}
+      labels:
+#           {{ range .pr.labels }}
+        - "{{ . }}"
+#           {{ end }}
+#     {{ end }}
+
+scms:
+  policy-repo:
+    kind: github
+    spec:
+      user: "{{ .github.git_author }}"
+      email: "{{ .github.git_email }}"
+      directory: "/tmp/policy-repo"
```

Comment:
You shouldn't set the directory where Updatecli clone git repositories.
If you run different pipelines from the same machine you will face unexpected as the same directory name may be used by different git repositories
 
```suggestion
```

By default Updatecli works from the directory `$TMP_DIR`/updatecli/
where $TMP_DIR depends on your operating system, linux machines will be /tmp

Reply:
Fair enough, will lean on `--clean=true` then for local testing.